tests/ops/qubit/test_attributes.py

```diff
@@ -79,3 +81,55 @@ def test_inclusion_after_addition(self):
     def test_tensor_check(self):
         """Test that we can ask if a tensor is in the attribute."""
         assert not qml.PauliX(wires=0) @ qml.PauliZ(wires=1) in new_attribute
+
+
+class TestSupportsBroadcasting:
+    """Test that all operations in the ``supports_broadcasting`` attribute
+    actually support broadcasting."""
+
+    def get_args_and_kwargs(self, name):
+        """Generate broadcasted parameters, wires and potential keyword arguments
+        for operations that support broadcasting; Batch size always is 3."""
```

Comment:
As compact as this is @dwierichs, I strongly recommend having separate self-contained classes per operation -- for future maintenance and readability!

Reply:
I split the attribute into operations with identical signature and grouped those together via `pytest.parametrize`. Then I included a test that the subsets actually contain all the operations that are in the attribute :) I hope that's nicer, it's definitely much more readable :D